README.md

```diff
@@ -468,7 +471,6 @@ class TritonPythonModel:
             if an_error_occurred:
               # If there is an error, the output_tensors are ignored
```

Comment:
Would it be clearer if we update the comment here to something like: `If there is an error, there is no need to pass the output_tensor to the InferenceResponse. The output_tensors that are passed in this case will be ignored.`?

Reply:
Agreed, updated.